modules/system.js

```diff
@@ -49,6 +50,8 @@ function System (cb, scope) {
 		this.minVersion = __private.minVersion;
 	}
 
+	constants.setConst('headers', __private);
```

Comment:
instead of create isolated `headers` constant, maybe will be better to add into `ws` object